demisto_sdk/commands/lint/commands_builder.py

```diff
@@ -231,8 +233,14 @@ def build_pylint_command(files: List[Path], docker_version: Optional[float] = No
     # disable xsoar linter messages
     disable = ['bad-option-value']
     # TODO: remove when pylint will update its version to support py3.9
-    if docker_version and docker_version >= 3.9:
-        disable.append('unsubscriptable-object')
+
+    if docker_version:
+        py_ver = parse(docker_version)
+        major = py_ver.major
+        minor = py_ver.minor
+
+        if major == 3 and minor == 9:
```

Comment:
what about 3.10?
keep the logic of "above" 3.9

Reply:
done